scripts/lint.js

```diff
@@ -22,11 +29,31 @@ const TESTS = {
       return invalidEntries;
     };
 
-    const invalids = icons.reduce(collector, []);
+    const invalids = simpleIconsData.icons.reduce(collector, []);
     if (invalids.length) {
       return `Some icons aren't in alphabetical order:
         ${invalids.map(icon => icon.title).join(", ")}`;
     }
+  },
+
+  /* Check the prettification of the data file */
+  prettified: function() {
+    const simpleIconsDataString = fs.readFileSync(
+      simpleIconsDataFile, "utf8").replace(/\r\n/g, '\n');
+    const simpleIconsDataPretty = `${JSON.stringify(simpleIconsData, null, "    ")}\n`;
+    if (simpleIconsDataString !== simpleIconsDataPretty) {
+      const dataDiff = diffLinesUnified(simpleIconsDataString.split("\n"),
+                                        simpleIconsDataPretty.split("\n"),
+                                        {
+                                          expand: false,
+                                          aColor: (v) => v,
+                                          bColor: (v) => v,
+                                          commonColor: (v) => v,
+                                          patchColor: (v) => v,
+                                          omitAnnotationLines: true
+                                        });
```

Comment:
> Not, if I omit the color fields is like if I pass the default values defined here. Where says "the default values are exactly as you provided them originally"? If I'm not lost, options object uses color settings by default.

Sorry for the confusion, I meant essentially the suggestion below. I tested this on Ubuntu and it seems to work as expected.

```suggestion
                                        {
                                          expand: false,
                                          omitAnnotationLines: true
                                        });
```

> I've replaced \r\n with \n in the prettified string. Can you try with this change?

Will try this later when I'm on my Windows boot again :upside_down_face: